lib/ts/recipe/dashboard/api/getTenantLoginMethodsInfo.ts

```diff
@@ -27,9 +27,11 @@ type TenantLoginMethodType = {
     tenantId: string;
     emailPassword: {
         enabled: boolean;
+        isThirdParty?: boolean;
```

Comment:
this is a bad name... have another object called thirdPartyEmailPasssword: {enabled: boolean}. Same goes for thirdpartypasswordless